zipline/finance/performance/tracker.py

```diff
@@ -385,11 +400,15 @@ def handle_market_close(self, dt, data_portal):
 
             benchmark_value = self.all_benchmark_returns[completed_session]
 
+            portfolio = self.get_portfolio(False)
+            expected_shortfall = portfolio.expected_shortfall
```

Comment:
Another reason to make this a method rather than a property is that this is expensive to call; `property` generally should only be used for attribute accesses that are computationally cheap.